tests/cnt-i18n_svg-direction/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,9 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
```

Comment:
For consistency with pkg-dir_* and #confreq-rs-svg-direction, I think this should be named cnt-svg-direction.

Remember, underscores are only for if you have multiple tests for one normative statement. (I realize that no one understands this and it's arbitrary and I would totally change it if we could, but too late!)